src/pages/code-for-good.js

```diff
@@ -0,0 +1,327 @@
+        width: 100%;
+    }
+`;
+
+const VesselAppContainer = styled('div')`
+    display: flex;
+    justify-content: space-between;
+    @media ${screenSize.upToMedium} {
+        flex-direction: column-reverse;
+    }
+`;
+
+const StyledVesselImage = styled('img')`
+    flex: 0 0 ${VESSEL_IMAGE_WIDTH}px;
+    width: ${VESSEL_IMAGE_WIDTH}px;
+    @media ${screenSize.upToMedium} {
+        width: 100%;
+    }
+`;
+
+const VesselImageDiv = styled('div')`
+    margin-top: ${size.xlarge};
+`;
+
+const StyledLogoImage = styled('img')`
+    margin-top: -${size.xlarge};
+    flex: 0 0 ${LOGO_IMAGE_WIDTH}px;
+    height: ${LOGO_IMAGE_HEIGHT}px;
+    width: ${LOGO_IMAGE_WIDTH}px;
+    @media ${screenSize.upToMedium} {
+        flex-direction: column-reverse;
+        height: 100%;
+        width: 100%;
+    }
+`;
+
+const StyledDashBoardImage = styled('img')`
+    flex: 0 0 ${DASHBOARD_IMAGE_WIDTH}px;
+    width: ${DASHBOARD_IMAGE_WIDTH}px;
+    @media ${screenSize.upToMedium} {
+        width: 100%;
+    }
+`;
+
+const StyledVesselCaption = styled(P3)`
+    color: ${colorMap.greyLightTwo};
+    margin-top: ${size.medium};
+    max-width: ${CAPTION_MAX_WIDTH};
+    @media ${screenSize.upToMedium} {
+        margin-bottom: ${size.large};
+        width: 100%;
+    }
+`;
+
+const VesselCaption = StyledVesselCaption.withComponent('figcaption');
+
+const StyledDashboardCaption = styled(P3)`
+    color: ${colorMap.greyLightTwo};
+    margin-bottom: ${size.xlarge};
+    margin-top: ${size.medium};
+    @media ${screenSize.upToMedium} {
+        margin-bottom: ${size.small};
+    }
+`;
+
+const DashboardCaption = StyledDashboardCaption.withComponent('figcaption');
+
+export default () => {
+    const metadata = useSiteMetadata();
+    const codeForGoodBreadcrumbs = [
+        { label: 'Home', target: '/' },
+        { label: 'Code for Good', target: '/code-for-good' },
+    ];
+
+    return (
+        <Layout>
+            <Helmet>
+                <title>Code for Good - {metadata.title}</title>
+            </Helmet>
+            <StyledHeroBanner
+                breadcrumb={codeForGoodBreadcrumbs}
+                background={BackgroundWaves}
+                fullWidth={true}
+                showImageOnMobile={false}
+            >
+                <Header>
+                    <div>
+                        <HeaderText>
+                            <ArticleH4>Code for Good</ArticleH4>
+
+                            <H3>Make a difference with MongoDB</H3>
+
+                            <StyledP>
+                                Use your current MongoDB skills and pick up new
+                                ones. Contribute back to O-FISH and help protect
+                                our oceans.
+                            </StyledP>
+                        </HeaderText>
+                        <StyledButton
+                            primary
+                            hasArrow={false}
+                            to="https://o-fish.github.io"
```

Comment:
ah! So with these links we have to use `href` and not `to`. The difference is with `to` it is a link to somewhere on the DevHub (or same-app) and `href` can be anywhere. Let's swap these external links to href!